### ISSUE TITLE
Enable sorting for categories in modal window

### ISSUE THREAD

**Issue by responseinformationsdesign** [fabarea/media#157]
System categories in modal window should be sortable (f. e. by title) or be displayed as category tree. Especially long unordered lists are very confusing.

**Comment by fabarea** [comment]
I agree, we have an open ticket [there](https://github.com/fabarea/vidi/issues/55) but we never finish it because it is not that simple. There is some starting code in this [PR](https://github.com/fabarea/vidi/pull/53/commits/b5c44a1b5f4593dfc1e48d731067c882f52100ff) if you feel like dusting the issue.
